### DISCUSSION TITLE
Duplicate scanner/remover

### DISCUSSION THREAD

**Discussion by shuvashish76** [VishnuSanal/Quotes#84] [Ideas]
Implement script or something to scan & remove duplicate  quotes from existing [Quotes.json](https://github.com/VishnuSanal/Quotes/blob/master/Quotes.json)...

**Comment by VishnuSanal** [comment]
> Implement script or something to scan & remove duplicate quotes from existing [Quotes.json](https://github.com/VishnuSanal/Quotes/blob/master/Quotes.json)...

@shuvashish76 As far as I know, there are no duplicates as of now. CMIIW.

**Reply by VishnuSanal** [reply]
I have created a script long while back to remove duplicates, I don't think still I have that with me! I think we can have a CI pipeline to test for duplicates
